### PR TITLE
Add Galton Board example

### PR DESCRIPTION
**Pull request by EnzioKam**
## PR Checklist

If you are contributing to `Javis.jl`, please make sure you are able to check off each item on this list:

- [ ] Did I update `CHANGELOG.md` with whatever changes/features I added with this PR?
- [x] Did I make sure to only change the part of the file where I introduced a new change/feature?
- [x] Did I cover all corner cases to be close to 100% test coverage (if applicable)?
- [x] Did I properly add Javis dependencies to the `Project.toml` + set an upper bound of the dependency (if applicable)?
- [x] Did I properly add test dependencies to the `test` directory (if applicable)?
- [x] Did I check relevant tutorials that may be affected by changes in this PR?
- [x] Did I clearly articulate why this PR was made the way it was and how it was made?

**Link to relevant issue(s)**

**How did you address these issues with this PR? What methods did you use?**
Adds the Galton Board example to the `Examples` page in the documentation.

### CODE REVIEW

**Review by TheCedarPrince — APPROVED** [review]
Hi @EnzioKam - thanks for the great PR! I think we are set to merge. I'll ping @Wikunia for a quick review but otherwise, this looks fantastic! The code is quite interesting - much smaller than I anticipated frankly and your documentation is stellar.

Ole, feel free to merge if it looks good to you!

File: examples/galton.jl [hunk 1]
```diff
@@ -49,31 +49,32 @@ function draw_pins(n, xpos, ypos, offset; color = "white")
 end
 
 """
-    flip_coin()
+    flip_coin(p)
 
-Simulates flipping a coin, returning -1 and 1 with equal probability. Helper function used in `move_ball`.
+Simulates flipping a coin, returning -1 with probability `p` and 1 with probability (1 - p). Helper function
```
Comment (Wikunia) [inline examples/galton.jl:54]: Should be the other way around. And `(1-p)` should also be in code tags.
You might want to have `flip_coin(p=0.5)` to have a reasonable default.

File: examples/galton.jl [hunk 2]
```diff
@@ -49,31 +49,32 @@ function draw_pins(n, xpos, ypos, offset; color = "white")
...
 
 """
-    move_ball(ball, first, last, offset, fno)
+    move_ball(ball, first, last, offset, p, fno)
 
 Moves `ball` from the `first` level of pins to the `last` level of pins (i.e. top to bottom). `offset` should
-be the same offset used when creating the bins using `draw_separators`. The value `fno` is the frame number
-from which the ball should start moving.
+be the same offset used when creating the bins using `draw_separators`. The value `p` is the probability that
+a ball will go left at each pin, and the value `fno` is the frame number from which the ball should start moving.
```
Comment (Wikunia) [inline examples/galton.jl:66]: I think it's the probability that it will go right (positive x direction), isn't it?

File: examples/galton.jl [hunk 3]
```diff
@@ -106,6 +107,7 @@ function galton(seed)
     height = 350
     offset = width / (n - 1)
     radius = offset / 4
+    p = 0.5
```
Comment (Wikunia) [inline examples/galton.jl]: Do you mind having this as a parameter like `seed`?

**Review by Wikunia — CHANGES_REQUESTED** [review]
Some small things. Thanks for the update

File: examples/galton.jl (hunk 1 above)
Comment (EnzioKam) [reply examples/galton.jl]: Yes it should 1 with probability `p`

File: examples/galton.jl (hunk 3 above)
Comment (EnzioKam) [reply examples/galton.jl]: I forgot about this previously, updated it now

**Review by Wikunia — APPROVED** [review]

**Comment by Wikunia** [comment]
Thank you :tada:
